agentliveness/cmd.py

```diff
@@ -22,6 +24,8 @@
 
 logger = logging.getLogger(__name__)
 
+DEBUG = True
```

Comment:
I assume that should be then `DEBUG = False`, so we do not log all the request with tokens in production.

Reply:
Yes. Thank you!